src/ui/QuantumDocument.vue

```diff
@@ -300,9 +300,13 @@ export default defineComponent({
   background-color: var(--color);
   --selected-background-color: rgba(68, 148, 202, 0.24);
   --selected-color: rgba(57, 131, 180, 0.459);
-  background-size: var(--grid-cell-size-x) var(--grid-cell-size-y);
+
+  background-size: var(--grid-cell-size-x) var(--grid-cell-size-y), var(--grid-cell-size-x) var(--grid-cell-size-y),
+    calc(var(--grid-cell-size-x) * 5) calc(var(--grid-cell-size-y) * 5), calc(var(--grid-cell-size-x) * 5) calc(var(--grid-cell-size-y) * 5);
   background-image: linear-gradient(to right, var(--grid-color) 1px, transparent 1px),
-    linear-gradient(to bottom, var(--grid-color) 1px, transparent 1px);
+    linear-gradient(to bottom, var(--grid-color) 1px, transparent 1px), linear-gradient(to right, rgba(71, 162, 223, 0.7) 1px, transparent 1px),
+    linear-gradient(to bottom, rgba(71, 162, 223, 0.7) 1px, transparent 1px);
```

Comment:
For the `rgba()` values, can you swap them for `var(--grid-color)`? This also makes the 'bold' lines a little more subtle and easier to see.

Reply:
I tried it out, this does look much nicer indeed